components/plurals/src/rules/runtime/ast.rs

```diff
@@ -33,6 +36,7 @@ pub enum Polarity {
 #[derive(Copy, Clone, Debug, PartialEq, Eq, Ord, PartialOrd)]
 #[repr(u8)]
 #[zerovec::make_ule(OperandULE)]
+#[allow(clippy::exhaustive_enums)] // this type is stable
```

Comment:
I wasn't really sure if this was stable @zbraniecki

Reply:
As noted above, `Operand` changes sometimes, but I don't know if it's one we want to generally make stable or non-stable

Reply:
I'd like to mark it as stable.